x-pack/plugins/infra/public/pages/metrics/inventory_view/components/ml/anomaly_detection/anomalies_table/anomalies_table.tsx

```diff
@@ -436,73 +455,77 @@ export const AnomaliesTable = (props: Props) => {
 
   return (
     <EuiFlexGroup direction="column">
-      <EuiFlexItem grow={1}>
-        <EuiSuperDatePicker
-          start={start}
-          end={end}
-          showUpdateButton={false}
-          onTimeChange={onTimeChange}
-          width="full"
+      {!hideDatePicker && (
+        <EuiFlexItem grow={false}>
+          <EuiSuperDatePicker
+            start={dateRange.from}
+            end={dateRange.to}
+            showUpdateButton={false}
+            onTimeChange={onTimeChange}
+            width="full"
+          />
+        </EuiFlexItem>
```

Comment:
Didn't want to mess with other components that use the anomalies table. We might want to refactor this component in the future and split content and date picker into separate components